doc/releases/release-notes-4.2.rst

```diff
@@ -517,6 +517,10 @@ New Drivers
    * :dtcompatible:`renesas,ra-wdt`
    * :dtcompatible:`silabs,siwx91x-wdt`
 
+* zbus
+
+   * :c:macro:`ZBUS_GLOBAL_ADD_OBS`
```

Comment:
I suppose declaring seems to be more suitable in this case since we are not adding that to an entity. 
```suggestion
   * :c:macro:`ZBUS_REGISTER_OMNI_OBS`
```

Reply:
The existing macro is `ZBUS_CHAN_ADD_OBS` to me it makes sense to re-use as much of the pattern as possible, rather than coming up with some new naming structure. `REGISTER` is not part of any ZBUS macros currently, whereas the proposed macro just switches `CHAN` for `GLOBAL`, indicating it is a global observer, rather than a channel observer.

Reply:
Agreed! Let's not add new things unnecessarily.